chain/network/src/peer_manager.rs

```diff
@@ -321,6 +375,30 @@ impl PeerManagerActor {
         }
     }
 
+    #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
+    fn start_routing_table_syncv2(
+        &self,
+        addr: Addr<Peer>,
+        ibf_set: Arc<Mutex<IbfSet<SimpleEdge>>>,
+    ) {
+        let ibf_vec =
+            ibf_set.lock().unwrap().get_ibf_vec(IbfSet::<SimpleEdge>::MIN_IBF_LEVEL as usize);
+        let seed = ibf_set.lock().unwrap().get_seed();
+        let _ = addr.do_send(SendMessage {
+            message: PeerMessage::RoutingTableSyncV2(RoutingSyncV2 {
+                version: 0,
```

Comment:
why is version 0?

Reply:
I changed it to 1. Should I change this to IbfVersion(int) ?

Reply:
I guess my question is more about what `version` is used for.

Reply:
It's unused currently, but it's for future use in case we want to add new fields. I would prefer to remove `version` if you are ok with it.

Reply:
@bowenwang1996 What do you think?